scipy-1.0/paper.tex

```diff
@@ -877,30 +873,28 @@ \subsubsection*{Sparse matrices}
 concerted effort to bring the \texttt{scipy.sparse} API into line with the
 equivalent NumPy API where possible.
 
+\subsubsection*{cKDTree}
+
+\input{ckdtree}
+
 \subsection*{Unified bindings to compiled code}
 
 \subsubsection*{LowLevelCallable}
 
 As of SciPy version 0.19, it is possible for users to wrap low-level functions
-in a \texttt{scipy.LowLevelCallable()} object that reduces the overhead for
-calling compiled C functions directly from Python.  Supported low-level
-functions include \texttt{PyCapsule} objects, ctypes function pointers, and
-cffi function pointers. The low level function signature must be consistent
-with the expectations of the routine it is passed to. For example, the
-documentation for \texttt{scipy.ndimage.generic\_filter} defines two acceptable
-C callback function signatures that may be used to produce functions that
-operate on each element of image data with low overhead. The C code may be
-generated using numba or Cython, for example, as long as the function call
-signature matches the specifications. Furthermore, it is even possible to
-generate a low-level callback function automatically from a Cython module using
-\texttt{scipy.LowLevelCallable.from\_cython}.
-
-\subsection*{Cython bindings for BLAS, LAPACK, and special}
+in a \texttt{scipy.LowLevelCallable} object that reduces the overhead of
+calling compiled C functions directly from Python.  
+Supported low-level functions include \texttt{PyCapsule} 
+objects, \texttt{ctypes} function pointers, and \texttt{cffi} function pointers. 
+Furthermore, it is even possible to generate a low-level callback function 
+automatically from a Cython module using \texttt{scipy.LowLevelCallable.from\_cython}.
+
+\subsubsection*{Cython bindings for BLAS, LAPACK, and special}
 
 SciPy has provided special functions and leveraged BLAS and
 LAPACK\cite{LAPACK} routines for many years. SciPy now additionally
-includes Cython\cite{behnel2011cython} wrappers for:
```

Comment:
The colon is not necessary here according to rule 2 of:
http://www.kentlaw.edu/academics/lrw/grinker/LwtaColons.htm